proposals/021-jsr-45.md

```diff
@@ -0,0 +1,38 @@
+# Complete SCP-11: implement JSR-45 for improved Scala debugging experience
+
+## Proposer
+
+Proposed by Ólafur Páll Geirsson, Twitter, December 2nd 2019.
+
+## Abstract
+
+This is a proposal to prioritize the completion of
+[SCP-11](https://github.com/scalacenter/advisoryboard/blob/master/proposals/011-debugging-symbols.md).
+To reduce the scope of SCP-11, this proposal suggests to focus only contributing
+JSR-45 support to the Scala compiler.
+
+[JSR-45](https://jcp.org/en/jsr/detail?id=45) is an approved specification that
+establishes standardized tools for correlating Java byte code to source code of
+languages other than the Java programming language. For example, the Kotlin
+compiler uses JSR-45 to improve the developer experience when working with
+debugging tools and code coverage tools.
+
+## Proposal
+
+The end goal of this proposal is to fix issue
+[scala/scala-dev#3](https://github.com/scala/scala-dev/issues/3). The Scala
+Center is expected to collaborate with the Scala compiler team to ensure that
+the JSR-45 implementation is well tested, documented and comprehensive (handles
+cases like inlined functions).
+
```

Comment:
It would be good to include a lot more motivation in this section: I've seen other summaries of the benefits of JSR-45 that would be good to include here.

Reply:
Good point, done.